src/http.c

```diff
@@ -727,7 +727,7 @@ int auth_log_in(struct tunnel *tunnel)
 		} else {
 			if (cfg->otp[0] == '\0') {
 				// Prompt for OTP token
```

Comment:
Perhaps change the comment as well?
`// Prompt for OTP token` → `// Prompt for 2FA token`

Reply:
```suggestion
				// Prompt for 2FA token
```

Reply:
Absolutely. Thanks.